dockerfiles/rclone_auto_config_dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM rclone/rclone:latest
+CMD printf "[$DRIVE_NAME]\ntype = drive\nscope = drive\ntoken = {\"access_token\":\"$TOKEN\",\"token_type\":\"Bearer\",\"refresh_token\":\"$REFRESH_TOKEN\",\"expiry\":\"$EXPIRY\"}" > /mnt/data/rclone.conf \
```

Comment:
Another possibility might be to `echo` the direct file stream as in the neuroconv auto config image

But there was one or two minor things preventing it from working right away